__tests__/app.test.js

```diff
@@ -91,3 +87,22 @@ describe("GET /api/users", () => {
     expect(res.body.user.length).toBe(4);
   });
 });
+
+describe("GET /api/articles", () => {
+  test("GET respond with an array of articles with the comment_count", async () => {
+    const res = await request(app).get("/api/articles").expect(200);
+
+    res.body.article.forEach(() => {
+      expect.objectContaining({
+        article_id: expect.any(Number),
+        title: expect.any(String),
+        topic: expect.any(String),
+        author: expect.any(String),
+        body: expect.any(String),
+        created_at: expect.any(String),
+        votes: expect.any(Number),
+        comment_count: expect.any(Number),
+      });
```

Comment:
Make sure to assert that the length is sufficiently large as well as the `forEach` - otherwise empty array will pass the test.